docs/get-started/bridging.mdx

```diff
@@ -42,8 +41,23 @@ The bridge uses these general steps:
 
 You can monitor the status of your bridge operations on the **Transaction History** tab.
 
-## Ghostnet
+## Bridging tokens on Ghostnet
 
 You can get Ghostnet tez from the [Ghostnet Faucet](https://faucet.ghostnet.teztnets.com/).
 
 Then you can use the [Testnet canonical bridge](https://bridge-xi.vercel.app/) to bridge those tokens to Etherlink or bridge tokens from Etherlink to Tezos.
+
+## How bridging works
+
+The bridging process uses two contracts, an EVM bridging contract that stores information about how many tokens different accounts have bridged and an exchanger contract that stores the tokens and issues tickets that represent those tokens.
```

Comment:
I don’t think that true: because we use tickets, there is no ledger at the smart contract level. I can be wrong, where did you get these information?

Reply:
This is based on Claude's article here: https://spotlight.tezos.com/moving-tez-etherlink. He also tells me that "The Etherlink rollup doesn’t mint, it receives tickets from the L1 and updates its internal ledger based on these tickets." I'll ask Claude if he can clarify.

Reply:
The sentence from Claude is true, but specifically in the doc quoted here you mention the smart contract on the L1, not the rollup.

Reply:
I see, I'm confused about what's on L2 and what's on L2. I'll confirm with Claude.